modules/swagger-codegen/src/main/resources/pistache-server/api-source.mustache

```diff
@@ -61,7 +74,7 @@ void {{classname}}::{{operationIdSnakeCase}}_handler(const Pistache::Rest::Reque
     {{/hasBodyParam}}{{#hasQueryParams}}
     // Getting the query params
     {{#queryParams}}
-    auto {{paramName}} = request.query().get("{{baseName}}");
+    auto {{paramName}} = ConvertQueryParam<{{dataType}}>(request.query().get("{{baseName}}"));
```

Comment:
So everything gets treated as an optional now?

Reply:
Oops yes you're right, I missed that,I'll correct that

Reply:
Oops, yes, I'll correct that.

Reply:
I'm returning back on this. request.query().get() allready returns an optional all the time. So it is allready treated as an optional at all times. Fixing this will involve changing pistache.

Reply:
```c++
namespace Uri {
    typedef std::string Fragment;

    class Query {
    public:
        Query();
        Query(std::initializer_list<std::pair<const std::string, std::string>> params);

        void add(std::string name, std::string value);
        Optional<std::string> get(const std::string& name) const;
        bool has(const std::string& name) const;

        void clear() {
            params.clear();
        }

    private:
        std::unordered_map<std::string, std::string> params;
    };
} // namespace Uri
```